src/SkySensorsAPI/ApplicationServices/WhetherStationAppService.cs

```diff
@@ -91,4 +93,16 @@ public async Task<IEnumerable<BasicWeatherStationDTO>> GetWeatherStationLists()
 		IEnumerable<WeatherStation> wsds = await wheatherStationRepository.GetWheaterStations();
 		return wsds.Select(w => WeatherStation.ToBasicWeatherStationDTO(w));
 	}
+
+	public async Task<bool> UpsertWeatherStation(WeatherStationBasicDTO weatherStationBasic)
+	{
+		bool j = await wheatherStationRepository.UpsertWeatherStation(weatherStationBasic.MacAddress, weatherStationBasic.GpsLocation.Longitude,  weatherStationBasic.GpsLocation.Latitude);
+		return j;
```

Comment:
Du behøves ikke at putte værdien ind i et variabel her.
```suggestion
		return await wheatherStationRepository.UpsertWeatherStation(weatherStationBasic.MacAddress, weatherStationBasic.GpsLocation.Longitude,  weatherStationBasic.GpsLocation.Latitude);
```